pkg/cnab/config-adapter/adapter.go

```diff
@@ -358,8 +358,10 @@ func (c *ManifestConverter) generateRequiredExtensions() []string {
 func lookupExtensionKey(name string) string {
 	key := name
 	// If an official supported extension, we grab the full key
-	// TODO: Do we want to error out here if unsupported?
-	if supportedExt, _ := extensions.GetSupportedExtension(name); supportedExt != nil {
+	supportedExt, err := extensions.GetSupportedExtension(name)
+	if err != nil {
+		// TODO: Issue linter warning
```

Comment:
When the porter manifest defines a required extension that is not supported by that version of porter, the linter should return a warning.

I'm using this issue to try out this feature in github where we can turn a pull request comment into an issue: https://help.github.com/en/github/managing-your-work-on-github/opening-an-issue-from-a-comment